packages/console/src/pages/TenantSettings/tabs/Settings/index.tsx

```diff
@@ -0,0 +1,22 @@
+import PageMeta from '@/components/PageMeta';
+import TabWrapper from '@/components/TabWrapper';
+
+import ProfileForm from './ProfileForm';
+import * as styles from './index.module.scss';
+
+type Props = {
+  isActive: boolean;
+};
+
+function Settings({ isActive }: Props) {
+  return (
+    <TabWrapper isActive={isActive} className={styles.tabContent}>
```

Comment:
This `TabWrapper` is only used on the sie page since the sie page needs to save data across all tabs.